backend/src/main/resources/application-azure-test.yaml

```diff
@@ -0,0 +1,19 @@
+spring:
+  profiles.include: okta-dev, auth-dev
+graphql.servlet.cors:
+  allowed-origins:
+    - https://simple-report-api-test.azurewebsites.net
+    - https://simple-report-test.azureedge.net
+    - https://test.simplereport.gov
```

Comment:
Does it actually need this? I think from `/app` to `/api` the browser won't even check.